### PR TITLE
Increase MTU tests timeout to remove flakes

### PR DESCRIPTION
**Pull request by coutinhop**
## Description

Increase timeouts on tests that verify MTU to 60s, due to the fact that Felix monitors host MTU every 30s. This should get rid of some flakiness in these tests.

## Todos

- [x] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: felix/fv/mtu_test.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ var _ = infrastructure.DatastoreDescribe("VXLAN topology before adding host IPs
 						Eventually(func() string {
 							out, _ := felix.ExecOutput("cat", "/var/lib/calico/mtu")
 							return out
-						}, "30s", "100ms").Should(ContainSubstring(vxlanEnabledMtu))
+						}, "60s", "100ms").Should(ContainSubstring(vxlanEnabledMtu))
```
Comment (tomastigera) [inline felix/fv/mtu_test.go:107]: I wonder if it would make sense to only check the condition every say 500ms or 1s given the huge timeout

**Review by tomastigera — APPROVED** [review]
with a nit that does not need a re-review if addressed

File: felix/fv/mtu_test.go (hunk 1 above)
Comment (coutinhop) [reply felix/fv/mtu_test.go:107]: that's probably a good idea, will do! thanks!

**Review by mgleung — APPROVED** [review]
LGTM, though we may want to keep note of these tests since increasing timeouts won't fix the issue if we end up running the tests on slower instances in the future.

**Pull request merged**